app/assets/javascripts/discourse/tests/unit/controllers/admin-site-settings-test.js

```diff
@@ -58,5 +63,13 @@ module("Unit | Controller | admin-site-settings", function (hooks) {
     results = controller.performSearch("digest", settings2);
     assert.deepEqual(results[0].siteSettings.length, 1);
     assert.deepEqual(results[0].siteSettings[0].setting, "digest_logo");
+
+    // ensures fuzzy search limiter doesn't limit too much
+    results = controller.performSearch("min length", settings2);
+    assert.deepEqual(results[0].siteSettings.length, 1);
+    assert.deepEqual(
```

Comment:
`deepEqual` should be used for comparing nested objects rather than primitives. (we can update the assertions above at a later time 😛)

```suggestion
    assert.strictEqual(results[0].siteSettings.length, 1);
    assert.strictEqual(
```

Reply:
Oh, ya, good call.